reconcile/utils/ocm/labels.py

```diff
@@ -108,6 +111,71 @@ class OCMAccountLabel(OCMLabel):
     account_id: str
 
 
+class LabelContainer(BaseModel):
+    """
+    A container for a set of labels with some convenience methods to work
+    efficiently with them.
+    """
+
+    labels: dict[str, OCMLabel]
+
+    def __len__(self) -> int:
+        return len(self.labels)
+
+    def get_label(self, name: str) -> Optional[OCMLabel]:
+        return self.labels.get(name)
+
+    def get_required_label(self, name: str) -> OCMLabel:
+        label = self.get_label(name)
+        if not label:
+            raise ValueError(f"Required label '{name}' does not exist.")
+        return label
+
+    def get_label_value(self, name: str) -> Optional[str]:
+        label = self.get_label(name)
+        if label:
+            return label.value
+        return None
+
+    def get_values_dict(self) -> dict[str, str]:
+        return {label.key: label.value for label in self.labels.values()}
+
+    def build_container_for_prefix(
```

Comment:
That's an odd pattern, saw class methods in other places in the Code. WDYT, makes a refactoring to a class method sense here?

Reply:
I'd prefer following the lead of `build_label_container` and create a regular function for this